charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -231,6 +231,16 @@ data:
                       container.args = container.command + container.args
                   container.command = ["tini", "--", "/entrypoint"]
                   pod.spec.containers[0] = container
+      {{ if .Values.houston.loggingSidecar.enabled }}
+                  log_cmd = "1> >( tee -a /var/log/{{ .Values.houston.loggingSidecar.name }}/out.log ) 2> >( tee -a /var/log/{{ .Values.houston.loggingSidecar.name }}/err.log >&2 )"
+                  if container.args[0:3] == ["airflow", "tasks", "run"]:
+                      container.command = ["tini", "--"]
+                      new_args = ["bash", "-c"]
+                      command_str = "/entrypoint " + ' '.join([str(arg) for arg in container.args]) + " " + log_cmd + "; curl -fsSL http://localhost:8000/quitquitquit"
```

Comment:
TODO likely http://localhost:8000/quitquitquit is the final endpoint we'll use

Reply:
Have added it as a config var instead. Are you talking of the endpoint name?

Reply:
@aliotta I don't understand your comment. Can you elaborate?

Reply:
@danielhoherd My comment is addressed by the config var. :-)

Reply:
oh yeah my orgional comment was missing a ket not in it whoops. Thanks for reading between the lines rishabh

Reply:
localhost:8000 is the endpoint because all containers share localhost, and the sidecar exposes its signaling port on localhost where airflow can POST to it.